helm/charts/hpe-csi-driver/templates/primera-3par-csp.yaml

```diff
@@ -68,6 +68,11 @@ spec:
           image: quay.io/hpestorage/alletra-9000-primera-and-3par-csp:v2.1.0
           {{- end }}        
           imagePullPolicy: {{ .Values.imagePullPolicy | quote }}
+          env:
```

Comment:
Please update the side car versions for the helm chart as well to match the yaml definitions

Reply:
done

Reply:
I don't see the sidecar versions in helm chart updated. @bhagyashree-sarawate can you help @maddiegos ?